django_app/redbox_app/redbox_core/consumers.py

```diff
@@ -93,25 +98,21 @@ async def handle_documents(self, message: CoreChatResponse, user: User) -> Seque
         doc_uuids: Sequence[UUID] = [UUID(doc.file_uuid) for doc in message.data]
         source_files = await self.get_files_by_core_uuid(doc_uuids, user)
         for source in source_files:
-            await self.send_to_client(
-                {
-                    "type": "source",
-                    "data": {"url": str(source.url), "original_file_name": source.original_file_name},
-                }
-            )
+            await self.send_to_client("source", {"url": source.url, "original_file_name": source.original_file_name})
         return source_files
 
     async def handle_text(self, message: CoreChatResponse) -> str:
-        await self.send_to_client({"type": "text", "data": message.data})
+        await self.send_to_client("text", message.data)
```

Comment:
this looks like a significant change? is this fixing a bug, or has send_to_client changed?

Reply:
It's just a small refactoring - `send_to_client()` has indeed changed. See line 112 below.

Reply:
thx for explantation